contrib/gofiber/fiber.v2/fiber.go

```diff
@@ -65,8 +66,8 @@ func Middleware(opts ...Option) func(c *fiber.Ctx) error {
 		}
 		span.SetTag(ext.HTTPCode, strconv.Itoa(status))
 
-		if cfg.isStatusError(status) {
-			// mark 5xx server error
+		if cfg.isStatusError {
```

Comment:
`cfg.isStatusError` is only set when the integration is configured. 
This will result in either every span being reported as error or every span reported as no error.